website/client/components/avatar.vue

```diff
@@ -1,6 +1,6 @@
 <template lang="pug">
 .avatar(:style="{width, height, paddingTop}", :class="backgroundClass", @click.prevent='castEnd()')
-  .character-sprites
+  .character-sprites(:style='{margin}')
```

Comment:
Yeah, that's tricky. Would it be sufficient to rename it to `inner-margin` or `sprites-margin`?